main.go

```diff
@@ -54,6 +54,7 @@ Options:
 	-top		Display summary table towards top.
 	-slow		Number of slowest tests to display. Default is 0, display all.
 	-nocolor	Disable all colors.
+	-pulse		Print "." every interval. Default is 0, disabled.
```

Comment:
Let's add `-pulse d` and jot down the unit (I know I'm going to forget in a week 😄)

```
Print "." every interval d, specified as a time.Duration. Default is 0, disabled.
```